EMS/admin-ui-bundle/assets/vite.config.ts

```diff
@@ -15,6 +15,7 @@ export default defineConfig({
       input: {
         action: 'src/action.js',
         app: 'src/app.js',
+        swaggerui: 'src/swaggerUi.js',
```

Comment:
I'm not a bug fan of uppercases in file name
```suggestion
        swaggerui: 'src/swagger-ui.js',
```

Reply:
renamed